src/utils/clickHandlers.js

```diff
@@ -7,7 +7,7 @@ import { updateAddOrRemoveCustomPaperOption, updatePaperSelectOptionsUnits } fro
 
 export function handleGenerateClick(generateEl, book) {
 	generateEl.setAttribute('disabled', true);
-	generateEl.innerText = 'Generating, this may take a little while...';
+	generateEl.innerText = 'Generating PDF...';
```

Comment:
appreciate the addition of `PDF`, but could you put back the warning of waiting -- it definitely can take a non-trivial moment for larger fic to finish processing and in the modern era of "everything is instant", seems useful to set expectations...

Reply:
I changed the text back to "Generating, this may take a little while..." but added a fontsize change so the text no longer resizes the button and overflows.